[2025-05-06T11:23:11.918+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2025-05-06T11:23:11.598261+00:00 [queued]>
[2025-05-06T11:23:11.923+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2025-05-06T11:23:11.598261+00:00 [queued]>
[2025-05-06T11:23:11.924+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-05-06T11:23:11.924+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-05-06T11:23:11.924+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-05-06T11:23:11.930+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2025-05-06 11:23:11.598261+00:00
[2025-05-06T11:23:11.932+0000] {standard_task_runner.py:55} INFO - Started process 15378 to run task
[2025-05-06T11:23:11.934+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2025-05-06T11:23:11.598261+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp95rdv9zz']
[2025-05-06T11:23:11.935+0000] {standard_task_runner.py:83} INFO - Job 20: Subtask is_api_available
[2025-05-06T11:23:11.961+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2025-05-06T11:23:11.598261+00:00 [running]> on host 081a24b4289c
[2025-05-06T11:23:11.988+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2025-05-06T11:23:11.598261+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-06T11:23:11.598261+00:00
[2025-05-06T11:23:11.989+0000] {http.py:120} INFO - Poking: api/
[2025-05-06T11:23:11.994+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2025-05-06T11:23:11.995+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2025-05-06T11:23:12.541+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2025-05-06T11:23:12.548+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20250506T112311, start_date=20250506T112311, end_date=20250506T112312
[2025-05-06T11:23:12.595+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-05-06T11:23:12.605+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
